count-on-click/style.css

```diff
@@ -0,0 +1,48 @@
+* {
+  background: rgb(227, 210, 210);
```

Comment:
In CSS, a color can be specified using a hexadecimal value in the form:
```#rrggbb```
https://www.w3schools.com/css/css_colors_hex.asp
(RGB in Hexadecimal)